igniter/bootstrap_repos.py

```diff
@@ -34,6 +34,20 @@
 LOG_ERROR = 3
 
 
+def sanitize_long_path(path, encoding=None):
```

Comment:
`encoding` doesn't seem to be used. Also I would add docstring to explain what it does and why. And one last thing, I would change a little bit the logic, so it bails out from the function if platfrom isn't windows. It should help to make it more readable.

Reply:
Not entirely sure what you mean by bailing out early to make it more readable, but I've tried something.